lua/multicursors/config.lua

```diff
@@ -141,6 +141,8 @@ local M = {
     hint_config = {
         border = 'none',
         position = 'bottom',
+        -- The following can also be used for further customization:
+        -- column_count = 1, -- number of columns to used when generating hints. Use 1 for a vertical display.
```

Comment:
I think it's better to move `column_count` to `generate_hints`, cause `hint_config` goes directly to hydra and it could confuses people.

Reply:
Yeah, I know that this was the original proposal in the issue, but I personally think it would be more confusing to have this in `generate_hints`, which is currently just a map from mode to generation handler.

What do you think of moving this to `generate_hints.config`? I think that will make it clearer that this corresponds to a setting for all generators (as represented by the outer keys).

Reply:
`generate_hints.config` seems good :+1: